owncloud/phoenix
# Add public link display

## Description
Adding public link display - following cases are implemented:
- [x] single file (same as folder)
- [x] single file with password (same as folder)
- [x] readonly folder
- [x] readonly folder with password
- [ ] full access folder
- [ ] full access folder with password
- [ ] upload only folder
- [ ] upload only folder with password

## Related Issue
- refs #1655 

## How Has This Been Tested?
- get https://github.com/owncloud/core/pull/35932
- setup the phoenix.baseUrl setting in the config.php - see https://github.com/owncloud/core/blob/bd3df85448af0e32bf199d6772b5b5eaf8c02091/core/routes.php#L84
- share a file publicly (old oc frontend for now)
- open this link in a private browser window
- see the browser opening up phoenix and displaying the file/image

## Screenshots (if appropriate):
![Screenshot from 2019-08-01 17-36-17](https://user-images.githubusercontent.com/1005065/62306887-f2935200-b482-11e9-9755-42bba884edba.png)
![Screenshot from 2019-08-01 12-30-17](https://user-images.githubusercontent.com/1005065/62306904-f9ba6000-b482-11e9-9651-064a11d13d48.png)


## Types of changes


## Checklist:



## Open tasks:
